build.gradle

```diff
@@ -65,7 +69,7 @@ task deploy(type: MavenExec) {
 // TODO should be dependent on build (but build needs to be incremental for that)
```

Comment:
why can't it be?

Reply:
todo is a remaineder of the previous editor :-|